src/react-apps/applications/service-development/src/config/routes.tsx

```diff
@@ -34,6 +36,7 @@ export const routes = [
       urlKey: 'shared.wip_link_github_url',
       imageSource: '../../designer/img/illustration-help-circle.svg',
       shadow: true,
+      iframeEndingUrl: 'Text',
```

Comment:
What is the meaning of this name: iframeEndingUrl?

Reply:
its the ending of the url to be used in the iframe, represents the path to the mvc:
https://dev.altinn.studio/designer/hannekot/ANewService/Text